tests/test_posixtestsuite_prejs.js

```diff
@@ -0,0 +1,11 @@
+// Copyright 2020 The Emscripten Authors.  All rights reserved.
+// Emscripten is available under two separate licenses, the MIT license and the
+// University of Illinois/NCSA Open Source License.  Both these licenses can be
+// found in the LICENSE file.
+
+(function () {
+  Module['onExit'] = function(status) {
+    console.log('POSIX test finished with status:' + status);
+    reportResultToServer(status);
```

Comment:
Its really a shame this doesn't work like this just out of the box for all browser tests.. but I know its complicated.